### PR TITLE
insert Unicode "Zero Width Space" to prevent error

### PR DESCRIPTION
**Pull request by GongT**
It will work with typescript compiler, and most editor/IDE supports UTF-8 (but not all, some will display a "invalid char").

#221

### CODE REVIEW

File: src/writeProcessor.ts [hunk 1]
```diff
@@ -73,6 +73,10 @@ export class WriteProcessor {
         return this;
     }
 
+    private protectComment(str: string): string {
```
Comment (horiuchi) [inline src/writeProcessor.ts]: Please add the comment how need this protection.

**Comment by horiuchi** [comment]
@GongT Thank you for your contribution. 👍 
I think that fix is needed.

Please fix Travis CI error.

**Comment by GongT** [comment]
emmm. I think this CI fail is caused by something wrong in "@types/commander"... what can I do now...

**Comment by svenefftinge** [comment]
Could this be merged? It is a blocker for me.

**Comment by horiuchi** [comment]
@GongT @svenefftinge Sorry late.
I fixed CI error and merged this PR.

**Pull request merged**